modoboa_installer/scripts/postwhite.py

```diff
@@ -67,3 +86,6 @@ def restore(self):
         if os.path.isfile(postwhite_backup_configuration):
             utils.copy_file(postwhite_backup_configuration, self.config_dir)
             utils.success("postwhite.conf restored from backup")
+
+    def download_file(self, url, destination):
```

Comment:
Why do you create a new function to download a file using urllib instead of using the function you're adding in the utils.py file?